### PR TITLE
ADCM-2351 Fixed update group config

### PR DESCRIPTION
**Pull request by a-alferov**

### CODE REVIEW

**Review by nagarin — APPROVED** [review]

**Comment by a-alferov** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @a-alferov! Build has been launched!

File: python/cm/models.py [hunk 1]
```diff
@@ -285,17 +285,20 @@ class ConfigLog(ADCMModel):
...
+                origin = {}
+            for key, value in group_keys.items():
+                if isinstance(value, Mapping) and key in renovator:
+                    origin[key] = update(
+                        origin.get(key, {}), renovator.get(key, {}), group_keys[key]
+                    )
                 else:
-                    origin[key] = value
+                    if value and key in renovator:
+                        origin[key] = renovator[key]
```
Comment (nagarin) [inline python/cm/models.py:301]: Seems like it has side-effect contradicting with line 293 - sometime you change object in place, sometime return new one

**Review by nagarin — CHANGES_REQUESTED** [review]

**Comment by a-alferov** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @a-alferov! Build has been launched!

**Comment by a-alferov** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @a-alferov! Build has been launched!

File: python/cm/models.py (hunk 1 above)
Comment (a-alferov) [reply python/cm/models.py]: fixed

**Review by nagarin — APPROVED** [review]

**Comment by a-alferov** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @a-alferov! Build has been launched!

**Comment by a-alferov** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @a-alferov! Build has been launched!

**Pull request merged**